_data/chains/eip155-777.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "cheapETH",
+  "chain": "cTH",
+  "network": "mainnet",
+  "rpc": [
+    "https://node.cheapeth.org/rpc"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Cheap Etherium",
+    "symbol": "cTH",
+    "decimals": 18
+  },
+  "infoURL": "https://cheapeth.org",
+  "shortName": "cTH",
+  "chainId": 777,
+  "networkId": 777,
+  "slip44": 60,
+  "ens": {
+    "registry":"0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e"
```

Comment:
are you sure about this?

Reply:
lol, honestly I didn't mean to PR it I wanted to see the CI test runs

Reply:
so i can close this?

Reply:
yes, apologies!